packages/runtime/datastore/src/test/localChannelContext.spec.ts

```diff
@@ -71,7 +74,7 @@ describe("LocalChannelContext Tests", () => {
 				dataStoreContext.baseLogger,
 				(content, localOpMetadata) => {},
 				(s: string) => {},
-				null as unknown as ISnapshotTree,
+				undefined as unknown as ISnapshotTree,
```

Comment:
Just for reviewer reference, this doesn't seem to be running at all, as far as I can tell `null` would make this blow just the same as `undefined` would when trying to access `tree.blobs` [here](https://github.com/microsoft/FluidFramework/blob/9220199ac126c77af867ef4b2148a63d452bf2d6/packages/runtime/datastore/src/localChannelContext.ts#L373), with a `null/undefined` `tree`, so this change seems ok.